pkgs/native_toolchain_c/test/helpers.dart

```diff
@@ -197,9 +197,6 @@ extension UnescapePath on String {
   String unescape() => replaceAll('\\', '/');
 }
 
-Future<String> readelfSymbols(String filePath) async =>
-    readelf(filePath, 'WCs');
-
 Future<String> readelfMachine(String path) async {
```

Comment:
Is this still used anywhere or can it be deleted?

Reply:
That's still used, the flags needed for that are available. (Was already used in cbuilder tests before as well.)